package.json

```diff
@@ -157,6 +160,7 @@
     "semver": "^7.5.3",
     "shelljs": "^0.8.2",
     "sinon": "^11.0.0",
+    "typescript": "^5.3.3",
```

Comment:
Why do we need TypeScript?

Reply:
Knip uses `typescript` and `@types/node` as peer dependencies to parse JavaScript and TypeScript files, build ASTs to find imports and exports, and as a foundation for module resolution. They're not regular dependencies of Knip as not to cause incompatibility issues with projects that do use TypeScript. Both `devDependencies` are already in `node_modules`, so they're not further impacting the size of the project.